docs/releases/2.0.txt

```diff
@@ -52,7 +52,7 @@ Simplified URL routing syntax
 The new :func:`django.urls.path()` function allows a simpler, more readable URL
 routing syntax. For example, this example from previous Django releases::
 
-    url(r'^articles/(?P<year>[0-9]{4})/$', views.year_archive),
+    url(r'^articles/(?P<year>[0-9]+)/$', views.year_archive),
```

Comment:
I found this confusing because just 'int' is used below which isn't bounded to 4 digits. I thought it best to make the regex url as close to the path.

Reply:
I guess it'd be better to describe how the new syntax may be less restrictive.

Reply:
I think changing the example to match is sufficient, it avoids distraction. We could maybe change the example entirely to use a model PK, which is a more common use case?

Reply:
Well, I think it's an important consideration when switching to the new syntax. I'll make a proposal.

Reply:
I agree it is, I just though the release note announcing the feature isn't necessarily the best place. Then again, maybe it is.